src/_layouts/default.html

```diff
@@ -41,7 +41,7 @@ <h1 class="mb-3">{{ page.title }}</h1>
           </section>
         </div>
 
-        <footer class="border-top bg-white px-3 py-2 flex-grow-0">
+        <footer class="border-top bg-white px-3 py-2 flex-grow-0 feedback-footer">
```

Comment:
where is the part where we're hiding by default and showing it if conditions are satisfied? i think we talked about this?

Reply:
FWIW, the argument I was trying to make when we talked about this in person was that this feedback widget isn't functional without JavaScript, so a noscript fallback is moot, and having it visible would technically make a button which clicking would have no effect.

Reply:
Ooh gotcha! That makes sense- thanks for clarifying